### PR TITLE
#19 codeception and one functional test

### PR DESCRIPTION
**Pull request by gozgunes**

### CODE REVIEW

**Comment by skafandri** [comment]
you mean #19 ?

File: codeception.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+suites:
+    functional:
+        actor: FunctionalTester
+        path: .
+        modules:
+            enabled:
+                - PhpBrowser:
+                    url: http://nginx
+                - \Tests\Functional\_support\Helper\Functional
```
Comment (kooliahmd) [inline codeception.yml:11]: What is the purpose of this line ?

Comment (gozgunes) [reply codeception.yml:11]: It doesn't have an use for now

Comment (skafandri) [reply codeception.yml:11]: If your PR is not final, please add a WIP to the PR title.

**Pull request closed without merging**